docs/labwc-actions.5.scd

```diff
@@ -80,6 +80,9 @@ Actions are used in menus and keyboard/mouse bindings.
 *<action name="ToggleAlwaysOnTop">*
 	Toggle always-on-top of focused window.
 
+*<action name="ToggleAlwaysBelow">*
+	Toggle always-below of focused window.
```

Comment:
Outside of the things mentioned below I don't really see any use-case for the action. Maybe some examples would be good here?
Something like "desktop" implementations or components (like resource monitors or "widgets") that don't support the wlr-layershell protocol. Or mentioning that the main use-case is for use in windowRules for those desktop components.

Not sure if it would make it more confusing though so maybe its fine as it is right now.

Reply:
Done.